beacon/src/region.rs

```diff
@@ -76,9 +76,9 @@ impl Default for Region {
 
 impl Region {
     pub fn from_i32(v: i32) -> Result<Self> {
-        ProtoRegion::from_i32(v)
+        ProtoRegion::try_from(v)
             .map(Self)
-            .ok_or_else(|| Error::unsupported_region(v))
```

Comment:
Did the method on Error disappear? if so can we fix that?

Reply:
That is the method

Reply:
Whoops, I see now. Brought back